src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotAddOrRemoveVirtualKeyword.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.DotNet.ApiCompatibility.Rules
     public class CannotAddOrRemoveVirtualKeyword : IRule
     {
         private readonly RuleSettings _settings;
+        private static bool isSealed(ISymbol sym) => sym.IsSealed || (!sym.IsVirtual && !sym.IsAbstract);
```

Comment:
nit: Private methods (and properties, which are also methods) are pascal cased in C#: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions.

Reply:
And another nit, we usually separate methods from fields via a new line.